cibuildwheel/windows.py

```diff
@@ -27,15 +27,17 @@
 
 
 def call(
-    args: Sequence[PathOrStr], env: Optional[Dict[str, str]] = None, cwd: Optional[str] = None
+    args: Sequence[PathOrStr], env: Optional[Dict[str, str]] = None, cwd: Optional[PathOrStr] = None
```

Comment:
Would have been nice with #672, would have fixed this for all OSs. :)

Reply:
haha! yeah... need to decide what to do with that...